testsuite/models/testrun.py

```diff
@@ -0,0 +1,80 @@
+''' Test run module.
+'''
+import json
+
+from testsuite.extensions import db
+
+
+class TestRun(db.Model):
+    ''' The results of a single test run.
```

Comment:
I don't have a good reason for putting them in separate tables, other than they *feel* like distinct objects. In particular snapshots, if we ever decided to start storing partial runs, that would be a pretty minor change at this point.

As for feature creep, I'm also planning on using this same db for https://github.com/sync-for-science/tracking/issues/76. I'm hoping that instead of passing the output along in a POST param, I'll be able to just pull it out of the database. Not 100% sure how it'll work yet.